page/sass/faq.scss

```diff
@@ -8,7 +8,7 @@
 
   .faq-content {
     padding-left: 10px;
-    @media (min-width: $screen-md) {
+    @media (min-width: $screen-md-min) {
```

Comment:
shoudnlt it me sm-max?

Reply:
no, its correct way - [link here](https://www.mtech-llc.com/blog/david-paredes/how-use-bootstrap-media-query-breakpoints)